src/app/components/CarouselCoins.tsx

```diff
@@ -41,7 +42,7 @@ export default function CarouselCoins({
     dispatch(getCoinData(currencyCode));
   }, [dispatch, currencyCode]);
 
-  if (isLoading) return <div>Loading...</div>;
+  if (isLoading || data.length === 0) return <CarouselCoinsSkeleton />;
```

Comment:
Let's say isLoading is false, the loading has finished and the final result in API call the data.length is 0. In this case also will you show the skeleton, I think should we show something like "No results found"?